vello/src/lib.rs

```diff
@@ -268,6 +277,13 @@ pub struct RenderParams {
     /// The anti-aliasing algorithm. The selected algorithm must have been initialized while
     /// constructing the `Renderer`.
     pub antialiasing_method: AaConfig,
+
+    /// Options for debug layer rendering.
+    ///
+    /// This only has an effect when the `debug_layers` feature is enabled.
+    // This is exposed publicly as a least-effort to avoid changing the API when features change.
+    // We expect the API to change here in the near future.
+    pub debug: DebugLayers,
```

Comment:
Since this is here, if this is set to anything but the feature is not enabled, then a warning should be printed.

Reply:
This isn't really intended to be used by end-user consumers, but I've added this message anyway